### ISSUE TITLE
Storing multitask labels

### ISSUE THREAD

**Issue by yngtodd** [yngtodd/datastore#2]
Create a common way to handle labels in multitask datasets.

**Comment by jacobhinkle** [comment]
I vote dataframe

**Comment by yngtodd** [comment]
haha Yeah. I was thinking about how multitask labels should be returned from `dataloaders`. 

```
# What should `label` be here:
for idx, (data, label) in enumerate(dataloader):
   ...
```

**Comment by yngtodd** [comment]
Added a minimal example of the proposed solution here: https://github.com/yngtodd/datastore/blob/multitask/notebooks/multitask.ipynb

**Comment by yngtodd** [comment]
Fixed with #3
